tasks.yaml

```diff
@@ -37,8 +37,7 @@ tasks:
         cmd: |
           set -e
           # uds zarf tools download-init does not work in 0.9.4 - https://github.com/defenseunicorns/uds-cli/issues/517
-          uds zarf package pull oci://ghcr.io/defenseunicorns/packages/init:v$(uds zarf version)
-          mv zarf-init-amd64-v$(uds zarf version).tar.zst zarf-init-amd64-$(uds zarf version).tar.zst
+          uds zarf package pull oci://ghcr.io/defenseunicorns/packages/init:$(uds zarf version)
```

Comment:
We had to explicitly add the `v` here for this to work on 0.9.4. I can't remember what 0.10.x version we tested but we had some issues with it due to the `v` and `oci://` issues if I recall right. Have you had success running this task on 0.10.x?

Reply:
Running validate locally fails with the explicit leading `v`, and succeeds with it removed.  
![image](https://github.com/defenseunicorns/uds-k3d/assets/7074946/9f4eaa5c-6005-42eb-9d92-028b2198f42c)

I'm running v0.10.4